test/no-functions.js

```diff
@@ -5,13 +5,17 @@ var Understudy = require('../').Understudy;
 var actor = new Understudy();
 
 actor.before('no-functions', function () {
-  assert.equal(true, false);
+  throw new Error('before shouldn\'t be reached');
 });
 
-actor.after('no-functions', function () {
```

Comment:
If you look at the file after changes, I deleted `actor.after` and modified `actor.before`.  I think `actor.after` is redundant in this case.